README.md

```diff
@@ -1,24 +1,208 @@
-# README
+<a name="readme-top"></a>
```

Comment:
_note: since there is no open PR in the frontend project to review, I'll leave my comments here:_

# Frontend comments

- [ ] There are multiple `console.log` statements and commented code all over the frontend project.  The instances seem to be too many to point out here.  Kindly, remove the debugging `console.log` statements and the commented code from the project.   That way, your repo will be cleaner. 👍 

![image](https://user-images.githubusercontent.com/57421823/219905990-204c3c12-ee69-44c0-9ffb-fee0064e5910.png)

- [ ]  On the "License" section of your frontend readme, the link is broken because it points to a file that does not exist in your project.

## Tips and extra help

Go [here](https://github.com/microverseinc/readme-template), grab the `MIT.md` file from that repo, paste it on the root of your project, and link to it in the "License" section of your readme.

That way, your app will be well documented 💪

- [ ] This piece of advice is just instructions for you, so you can include a proper License.  After doing so, kindly remove this text:

![image](https://user-images.githubusercontent.com/57421823/219906066-c889e659-11e0-495b-8402-f05757e54b4b.png)

- [ ] Your backend project needs to be run on the port 3001 which is not the default.  Kindly, add those instructions in your readme about how to properly run your project.

- [ ] On your readme, the featured instructions do not indicate how to properly run the frontend project.  Backend instructions are present, but no instructions for the frontend.  Kindly fix the instructions so your users are able to run the frontend project.

- [ ]  Please, kindly refactor this and all other functions in your project to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered non-best practice.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry:

![image](https://user-images.githubusercontent.com/57421823/219906461-52b92814-5781-46d7-bc52-fc8497db61ea.png)

- [ ] When creating and selecting a model, I can't view (or find) a details page.  Kindly add one as this is part of the requirements.

Reply:
Thanks @Meltrust  for the constructive feedback. I have already implemented all your requested changes.